omod-1.9/src/main/java/org/openmrs/module/webservices/rest/web/v1_0/resource/openmrs1_9/VisitResource1_9.java

```diff
@@ -118,6 +116,21 @@ public String getDisplayString(Visit visit) {
 		return visit.getActiveAttributes();
 	}
 	
+	@PropertyGetter("encounters")
+	public Collection<Encounter> getActiveEncounters(Visit visit) {
```

Comment:
to make this a little clearer, I would just call it "getNonVoidedEncounters"

Reply:
makes sense. I just renamed the method. Thanks.